### ISSUE TITLE
Bug - Save image as PDF file... - Linux (Praat 6.0.42)

### ISSUE THREAD

**Issue by rolandomunoz** [praat/praat#745]
If I use `Select outer viewport: 5.5, 12, 0, 4` in the Praat Picture and save that part as a PDF file, the resulting image only contains the upper left part `Select outer viewport: 0, 6, 0, 4` of the Praat Picture.

**Comment by no-buddy** [comment]
I run this script...

```
Create Sound from formula: "10Hz", 1, 0, 12, 44100, "sin(2*pi*10*x)"
Erase all

for i from 0 to 11
	j=i+1
	Select outer viewport: 'i', 'j', 0, 4
	Draw: 'i', 'j', 0, 0, "yes", "Curve"
endfor

Select outer viewport: 6, 12, 2, 6

Save as PDF file: "/tmp/10Hz.pdf"
Remove
```

and reproduced the problem on Linux/Debian.
Use 0,0 as upper left corner of your selection till this si sorted...
I always did that in the past as a default and did not stumble upon this problem. :)

**Comment by PaulBoersma** [comment]
Solved for version 6.1.53. Thanks for the explicit explanation.
